frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseRebaseBranchOntoParentAction.java

```diff
@@ -58,52 +59,56 @@ public void update(AnActionEvent anActionEvent) {
 
     if (state.isEmpty()) {
       presentation.setEnabled(false);
-      presentation.setDescription("Can't rebase due to unknown repository state");
+      presentation.setDescription(GitMacheteBundle.message("action.rebase.description.disabled.git4idea.unknown-state"));
 
     } else if (state.get() != Repository.State.NORMAL) {
 
       var stateName = Match(state.get()).of(
-          Case($(Repository.State.GRAFTING), "during an ongoing cherry-pick"),
-          Case($(Repository.State.DETACHED), "in the detached head state"),
-          Case($(Repository.State.MERGING), "during an ongoing merge"),
-          Case($(Repository.State.REBASING), "during an ongoing rebase"),
-          Case($(Repository.State.REVERTING), "during an ongoing revert"),
+          Case($(Repository.State.GRAFTING), GitMacheteBundle.message("git4idea.repository.state.ongoing.cherry-pick")),
```

Comment:
I don't think these keys should include any mention of `git4idea`, esp. for the things that aren't git4idea-specific like repository state

Reply:
good point